bin/cyhy-ip

```diff
@@ -163,6 +162,15 @@ def add(db, owner, cidrs):
     intersections = db.RequestDoc.get_all_intersections(
         cidrs
     )  # intersections from database
+    geo_loc_db = GeoLocDB()
+    has_restricted_ips = False
+    for cidr in cidrs.iter_cidrs():
+        if geo_loc_db.check_restricted_cidr(cidr):
+            has_restricted_ips = True
```

Comment:
Explanatory comments should be here as well since the reason is the same.

```suggestion
            # We can't just exit here since check_restricted_cidr()
            # prints output for each restricted CIDR block and we
            # want the user to see all restricted CIDR blocks
            # without having to run multiple times.
            has_restricted_ips = True
```